doc.md

```diff
@@ -0,0 +1,59 @@
+# xo-test [![Build Status](https://travis-ci.org/vatesfr/xo-test.png?branch=master)](https://travis-ci.org/vatesfr/xo-test)
+
+> Test client for Xo-Server
+
+<a>
+
+## Organization
+
+Reminder: the commands are composed of a **component** and an *action* (**user**.*create*, **job**.*getAll*, **vm**.*delete*,...). 
+
+- The tests are classified by component as shown below :
+```
+src
+├─ [component1]
+|     ├─ __snapshots__
+|     |     └─ [component1].spec.js.snap 
+|     └─ [component1].spec.js
+├─ [component2]
+¦     └─ [component2].spec.js
+¦
+├─ _xoWithTestHelpers.js
+└─ util.js
+```
+
+- In each file `[component].spec.js`, you should have :
+```
+describe("[component]", () => {
+  describe(".[action] :", () => {
+    it("what the test does...", () => {
+      // some tests
+    });
+    it("what the test does...", () => {
+      // some tests
+    });
+  });
+});
+```
+
+- There is a helper `_xoWithTestHelpers.js` to manage the temporary items created for the tests (vm, backup, user...). 
+
+
+## Adding a test
+
+- First, check if the main command `[component].[action]` used by the feature you want to test, is already tested.
+  - If it is : write your test in this command's `describe()`. 
+  - If it is not the case :
+    1. Go to the `[component]` folder if it exists, otherwise create it.
+    2. Open the `[component].spec.js` file if it exists, otherwise create it.
+    3. And write your test respecting the organization of a `[component].spec.js` described above.   
+
+- Make a sentence for the title of the test. It must be clear and consistent. For example: 
+  `it("creates a backupNg job without schedule", () => { /* some tests */ });`
+  `it("fails trying to run a backupNg job without schedule", () => { /* some tests */ });`
+
+- If the feature you want to test is not implemented : write it and skip it, using `it.skip()`.
+
+- The test must be simple : one feature by test.
+
+- Take values ​​that cover the maximum of testing possibilities.
```

Comment:
Please add a section which explain how to run tests.

Reply:
There is already a `#Usage` part to know how to run tests in the `README`.